README.md

```diff
@@ -2,55 +2,75 @@
 
 # Project Name
 
-> Description the project.
+> Pokemon Website.
 
+## Description
 
-## Built With
+- This project is built using PokeAPI for pokemons and Involvement API for interactions.
+- In this project you can view different Pokemons and their abilities.
+- You can give a like to your favourite pokemons.
+- You can also share your thoughts on the pokemon by leaving comments on it.
 
-- Major languages
-- Frameworks
-- Technologies used
+## Built With
 
-## Live Demo (if available)
+- HTML, CSS, JavaScript
+- Webpack
+- Jest
+- RESTFUL APIs
 
-[Live Demo Link](https://livedemo.com)
+### Prerequisites
 
+- Vistual Studio Code
+- Browser (Firefox / Google Chrome)
+- Node.js (v6.11.2+)
+- linters
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To get a local copy up and running follow these simple example steps.
 
+## Install Dependencies(with following Command)
 
-To get a local copy up and running follow these simple example steps.
+- npm install --save -dev
 
-### Prerequisites
+## Build webpack
+
+- run command npm run build
 
-### Setup
+## Start server
 
-### Install
+- npm start
+
+## To test the app run following command
+
+- npm test
 
 ### Usage
 
-### Run tests
+Showcase Pokemons and their abilites. You can like and post comments on each pokemon.
+
+## Feature Video
 
-### Deployment
+Click this [Video](https://drive.google.com/file/d/1a7RdEyUAWa7g6i99kc4qzchwkugp2BVP/view?usp=sharing) to see a descriptive video about the features of the website 
 
+## Live Demo
 
+Click here to see the [Live Demo](https://shella12.github.io/Pokemon.github.io/dist/) of the website 
+
+![](./app_screenshot.png)
 
```

Comment:
**[Optional]**

- [ ] It would be nice if you add screenshots of your projects here to make your readme more descriptive :blush: